react/package.json

```diff
@@ -34,7 +35,28 @@
     "react-dom": "^16.2.0",
     "react-intl": "^2.4.0",
     "react-test-renderer": "^16.2.0",
-    "typescript": "^3.3.4000"
+    "typescript": "^3.3.4000",
+    "vtex.flex-layout": "http://vtex.vteximg.com.br/_v/public/typings/v1/vtex.flex-layout@0.4.2/public/_types/react",
+    "vtex.product-kit": "http://vtex.vteximg.com.br/_v/public/typings/v1/vtex.product-kit@1.9.1/public/_types/react",
```

Comment:
Remove those changes made by toolbelt. Update your toolbelt and it shouldn't do that anymore.